app/src/app/home/home.component.sass

```diff
@@ -14,10 +14,11 @@ a
     text-decoration: underline
 
 div#logos
-  display: flex
+  display: inline-flex
   margin-top: 2em
-  justify-content: center
   align-items: center
+  justify-content: space-around
```

Comment:
justify-content has no effect now because there is no remaining width space to distribute. Inline element have no width constraint by default.